lib/drivers/abbottFreeStyle.js

```diff
@@ -409,13 +409,16 @@ module.exports = function (config) {
           .done();
         dataToPost.push(smbg);
       } else if (datum.readingType === 'ketone') {
-        // for now we don't post these but we'll want to someday
+        // These meters store ketone values as 18 * the mmol/L value of the ketones
+        // Because this is specific to these devices, we convert back
+        // in the driver and upload the data as mmol/L.
+        var ketoneValue = datum.reading / 18.0;
```

Comment:
Aren't we going to get some floating point oddness? Does the spec give any info on significant digits of ketone readings?